scripts/semver.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+###
+# semantic version comparition using semver specification http://semver.org/
+# This bash script compares pre-releases alphabetically as well
+#
+# returns 1 when A greater than B
+# returns 0 when A equals B
+# returns -1 when A lower than B
+#
+# Usage
+# chmod +x semver.sh
+# ./semver.sh 1.0.0 v1.0.0-rc.0
+# --> 1
+#
+# Author Ariel Rodriguez
+# License MIT
+###
+semver_compare() {
+  local version_a version_b pr_a pr_b
+  # strip word "v" and extract first subset version (x.y.z from x.y.z-foo.n)
```

Comment:
As I see it, you have copied this script from somewhere. Still, I’d capitalise the first letters of all the comments.

And add the definite article (`the`) before ordinal number `first`.